text_collector_examples/iptables_collector.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import subprocess
+import re
+import os
+
+# Change to your collector.textfile.directory
+prom_outfile = '/var/lib/prometheus/node-exporter/iptables_collector.prom'
```

Comment:
Textfile collector examples should simply print to STDOUT. This way they're easy to test, and you can use `sponge` to atomic write the `.prom` files.

Reply:
Easy to test, but harder to use. Instead of, modifing the code, you could redirect prom file to /tmp and `cat` the file for debug.

Reply:
Sorry, but this is a requirement for these examples.

Reply:
Fixed with https://github.com/prometheus/node_exporter/pull/1264/commits/0e48c29311f58a8417b9288152300ae66a2baeb2 ;)

Reply:
It's not fixed, its' still writing a file. We can only include it if you remove that.